fiftyone/brain/internal/core/redaction.py

```
@@ -0,0 +1,426 @@
+_STACK_BLUR_KERNEL_SIZE = lambda w: min(w, 50) * 2 + 1  # ensure ksize is odd
+
+
+def create_redaction(
+    samples,
+    label_field,
+    label_classes,
+    redaction_type,
+    redaction_method,
+    redaction_field,
+    remote_redacted_media_dir,
+    progress,
```

Ruff (0.14.11)

Unused function argument: `progress`

(ARG001)